app/views/shared/_navbar.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="navbar-wagon">
   <!-- Logo -->
   <%= link_to root_path, class: "navbar-wagon-brand" do %>
-    <%= image_tag "logo.png", height: 50 %>
+    <%= image_tag "https://image.flaticon.com/icons/png/512/206/206881.png", height: 50 %>
```

Comment:
check in browser

Reply:
[https://image.flaticon.com/icons/png/512/206/206881.png](url)